UXWComponent.js

```diff
@@ -0,0 +1,39 @@
+function UXWComponent(props) {
+  return (
+    <div className="container">
+      <div className="flex flex-row">
+        <div className="flex flex-col">
+          <p className="pb-4 max-w-prose">
+            Components are reusable blocks of code. Components helps building UXW’s products efficiently and consistancy.
+          </p>
+        </div>
+        <div className="flex-grow"></div>
+      </div>
+      <p className="pb-4 max-w-prose">
+        To use this components, you need to include <span className="uxw-code">uxwcore.css</span> and <span className="uxw-code">uxw.css.</span>
+        Read more about <RouterLink to="/introduction" target="_blank">Installation and Usage </RouterLink>to learn how this works.
+      </p>
+      <div className="flex max-w-prose p-4 my-4 uxw-bg-steel-2 rounded-lg border-black rounded-md shadow-inner"><p className="max-w-prose">Use CSS class names on your HTML elements to apply the style. You don’t need to write CSS once you included <span className="uxw-code">uxwcore.css </span> and <span className="uxw-code">uxw.css</span> on your file.</p></div>
+      <div className="max-w-7xl grid grid-cols-3 gap-8 my-10">
+        <RouterLink to="/colorclass">
```

Comment:
When you click this, the left side probably won't be set properly. You'll probably need to pass the set function into UXWComponent, but I wouldn't worry about that for now.

Reply:
Good catch. I will write it down for future fixing